english/metallica/metallica.md

```diff
@@ -1,8 +1,5 @@
-Metallica Wiki? [here](https://en.wikipedia.org/wiki/Metallica)
+You are in for a treat!
 
-The Metallica website [here](https://www.metallica.com/)
-
-Metallica Twitter [here](https://twitter.com/Metallica)
-
-Metallica YouTube Channel [here](https://www.youtube.com/user/MetallicaTV)
+Brief history (Wikipedia): Metallica is an American heavy metal band. The band was formed in 1981 in Los Angeles, California, by drummer Lars Ulrich and vocalist/guitarist James Hetfield, and has been based in San Francisco, California for most of its career. The group's fast tempos, instrumentals and aggressive musicianship made them one of the founding "big four" bands of thrash metal, alongside Megadeth, Anthrax and Slayer. Metallica's current lineup comprises founding members Hetfield and Ulrich, longtime lead guitarist Kirk Hammett and bassist Robert Trujillo. Guitarist Dave Mustaine (who went on to form Megadeth) and bassists Ron McGovney, Cliff Burton and Jason Newsted are former members of the band. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.